src/utils/params-to-actions.util.ts

```diff
@@ -78,7 +78,7 @@ export const paramsToPendingActions = (
               address: preview.approveTo
             },
             entrypoint: 'approve',
-            amount: preview.amount
+            amount: `-${preview.amount}`
```

Comment:
why there is no such check as above? 
```
preview.amount === '0' ? '0' : `-${preview.amount}`
```
It could be great to create and reuse some fn here